### PR TITLE
allow setting column to false to ignore it in addTimestamps

### LINKED ISSUE #1985: Allow addTimestamps to skip one of the columns
In some scenarios, the developer may not want both timestamp columns. I instinctively tried to supply `false`, but it results in an error caused by an empty field name.

My suggestion is either to:
- throw an error when a non-string/null value is supplied to either field names
- threat `false` as "skip this column" (and, of course, throw an error if both are false)

I would greatly support the latter (and probably provide a PR if greenlighted) since I see that happening a lot around my current project. A simple example: a logging table never gets updated entries, so `updated_at` makes no sense. The current method to achieve that is calling `addColumn()` by hand.

### PR DESCRIPTION
**Pull request by MasterOdin**
Closes #1985 

This allows using the boolean `false` for the first two arguments to `addTimestamps` and `addTimestampsWithTimezone` to have it skip adding that column to the table.

### CODE REVIEW

File: tests/Phinx/Db/TableTest.php [hunk 1]
```diff
@@ -136,6 +141,72 @@ public function testAddTimestamps(AdapterInterface $adapter, $createdAtColumnNam
...
+
+        $columns = [];
+
+        foreach ($actions as $action) {
+            $columns[] = $action->getColumn();
+        }
+
+        $this->assertEquals(1, count($columns));
+
+        $this->assertEquals('created_at', $columns[0]->getName());
```
Comment (dereuromark) [inline tests/Phinx/Db/TableTest.php:163]: We should use assertSame() for scalar values where possible.

File: tests/Phinx/Db/TableTest.php [hunk 2]
```diff
@@ -136,6 +141,72 @@ public function testAddTimestamps(AdapterInterface $adapter, $createdAtColumnNam
...
+        $this->assertEquals('timestamp', $columns[0]->getType());
+        $this->assertEquals('CURRENT_TIMESTAMP', $columns[0]->getDefault());
+        $this->assertEquals(false, $columns[0]->getTimezone());
```
Comment (dereuromark) [inline tests/Phinx/Db/TableTest.php:166]: assertFalse()

**Comment by igorsantos07** [comment]
Damn, I don't remember the last time I created an issue in an OS repo and the idea got accepted AND done in less than a few months. I didn't even have time to get psychologically prepared to make my "first" PR into the project :(((

Congratz you all! hahaha

File: tests/Phinx/Db/TableTest.php (hunk 1 above)
Comment (MasterOdin) [reply tests/Phinx/Db/TableTest.php]: changed.

File: tests/Phinx/Db/TableTest.php (hunk 2 above)
Comment (MasterOdin) [reply tests/Phinx/Db/TableTest.php]: changed.

**Pull request merged**